test/remote/gateways/remote_adyen_test.rb

```diff
@@ -363,6 +363,30 @@ def test_successful_purchase_with_risk_data
     assert_equal '[capture-received]', response.message
   end
 
+  def test_successful_purchase_with_splits
+    split_data = [{
+      'amount' => {
+        'currency' => 'USD',
+        'value' => 50
+        },
+      'type' => 'MarketPlace',
+      'account' => '163298747',
+      'reference' => 'QXhlbFN0b2x0ZW5iZXJnCg'
+    }, {
+      'amount' => {
+        'currency' => 'USD',
+        'value' => 50
+        },
+      'type' => 'Commission',
+      'reference' => 'THVjYXNCbGVkc29lCg'
+    }]
+
+    options = @options.merge({ splits: split_data })
+    response = @gateway.purchase(@amount, @credit_card, options)
+    assert_failure response
+    assert_match 'Invalid additionalData: MarketPlace account is not configured for this merchant', response.message
```

Comment:
The Spreedly test account is not set up to allow MarketPlace transactions, which is what splits are. 

From [Adyen's documentation](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/v50/payments__reqParam_splits):
> Splits: The details of how the payment should be split when distributing a payment to a MarketPay Marketplace and its Accounts.

Therefore, I wrote the remote test to fail when passing in splits with the response that it's not configured, which at least confirms that the field was successfully passed to the gateway. 

I'm open to suggestions on improving this, including just deleting this test altogether.

Reply:
🤔 Would we need to set up a new test account to have this enabled?

Reply:
I'm not sure. Is it worth requesting that Adyen updates our account / adds a new account with this feature? The requested deadline for this from our customer is Friday.

Reply:
In general, it seems like a successful test should ... well, succeed? So if we can't actually test that, I'm in favor of getting rid of it (and maybe creating a GH issue to fill in the test that someone someday might pick up.)

I think there may be a bigger conversation to have here about test fixtures -- it seems like it happens not infrequently that account-side modifications need to be made that might not be reflected in the setup of every AM developer ever. That is definitely beyond the scope of this add, though :D

Reply:
That makes sense. Remote tests are not ideal in many ways...

I've gone ahead and deleted this remote test for now. We definitely don't have complete test coverage for all of the GSFs in Active Merchant, so I don't think it's critical that we update our Adyen gateway integration just for this.